react/features/invite/functions.js

```diff
@@ -397,3 +398,39 @@ export function searchDirectory( // eslint-disable-line max-params
                 return Promise.reject(error);
             });
 }
+
+/**
+ * Returns descriptive text that can be used to invite participants to a meeting
+ * (share via mobile or use it for calendar event description).
+ *
+ * @param {string} inviteUrl - The conference/location URL.
+ * @param {boolean} includeDialInfo - Whether to include or not the dialing
+ * information link.
+ * @param {boolean} useHtml - Whether to return html text.
+ * @returns {string}
+ */
+export function getShareInfoText(
+        inviteUrl: string, includeDialInfo: boolean, useHtml: ?boolean) {
+    let roomUrl = inviteUrl;
+
+    if (useHtml) {
+        roomUrl = `<a href="${roomUrl}">${roomUrl}</a>`;
+    }
+
+    let infoText = i18next.t('share.mainText', { roomUrl });
+
+    if (includeDialInfo) {
+        const { room } = parseURIString(inviteUrl);
+        const href = inviteUrl.substring(0, inviteUrl.indexOf(room));
+        let dialInfoPageUrl = `${href}static/dialInInfo.html?room=${room}`;
```

Comment:
Maybe this logic can be combined with what exists in InfoDialog and pulled into a separate helper?